cuda/common/include/pcl/cuda/common/eigen.h

```diff
@@ -233,7 +233,7 @@ namespace pcl
       //Scalar scale = mat.cwiseAbs ().maxCoeff ();
       float3 scale_tmp = fmaxf (fmaxf (fabs (mat.data[0]), fabs (mat.data[1])), fabs (mat.data[2]));
       float scale = fmaxf (fmaxf (scale_tmp.x, scale_tmp.y), scale_tmp.z);
-      if (scale <= FLT_MIN)
+      if (scale <= std::numeric_limits<float>::min())
```

Comment:
Not sure about the author's intent.

Reply:
`scale` shouldn't be close to zero because of the division below, so `min()` should be correct